src/redisraft.c

```diff
@@ -1756,6 +1759,10 @@ static void handleInfo(RedisModuleInfoCtx *ctx, int for_crash_report)
     RedisModule_InfoAddFieldULongLong(ctx, "proxy_failed_reqs", rr->proxy_failed_reqs);
     RedisModule_InfoAddFieldULongLong(ctx, "proxy_failed_responses", rr->proxy_failed_responses);
     RedisModule_InfoAddFieldULongLong(ctx, "proxy_outstanding_reqs", rr->proxy_outstanding_reqs);
+
+    RedisModule_InfoAddSection(ctx, "stats");
+    RedisModule_InfoAddFieldULongLong(ctx, "appendreq_received", rr->appendreq_received);
```

Comment:
Not sure about naming, I'm open to suggestions. 

We should expand this pattern to other messages over time. Stats for these might be useful:

appendreq
appendresp
snapshotreq
snapshotresp
reqvotereq
revqoteresp